main.go

```diff
@@ -177,759 +145,51 @@ func main() {
 //   2. Create database
 //   3. Run data conversion
 //   4. Generate report
-func toSpanner(driver, projectID, instanceID, dbName string, ioHelper *ioStreams, outputFilePrefix string, now time.Time) error {
+func toSpanner(driver, projectID, instanceID, dbName string, ioHelper *conversion.IOStreams, outputFilePrefix string, now time.Time) error {
```

Comment:
The naming of this function and its comment are now stale.

How about renaming this function to "commandLine" since it essentially provides the commandLine functionality of HarbourBridge.

Also, suggest updating the function comment:

// commandLine provides the core processing for HarbourBridge
// when run as a command-line tool. It performs the following steps:
//   1. Run schema conversion
//   2. Create database
//   3. Run data conversion
//   4. Generate report

Reply:
renamed function and updated comment